scripts/manage_demux_stats_thresholds.py

```diff
@@ -23,6 +24,7 @@ def __init__(self, instrument, chemistry, paired, read_length):
             self.paired = paired
             self.read_length = read_length
             self.set_undet_indexes_perc()
+            self.set_correction_factor_for_sample_in_pool()
```

Comment:
Hej @chuan-wang I don't know why we need a function for this (I believe you tried to be consistent with already existing style). But looking at the logic I think you can remove the 2 functions `set_undet_indexes_perc` and `set_correction_factor_for_sample_in_pool` and directly set the values here like 

```
self.undet_indexes_perc = 5
self.correction_factor_for_sample_in_pool = 0.75
```
also can **remove** the lines in the beginning

```
self.undet_indexes_perc = None
self.correction_factor_for_sample_in_pool = None
```